schedule_lessons/templates/dashboard/schedule_lesson.html

```diff
@@ -162,7 +166,10 @@ <h4 class="edit-availability-subtitle">Details</h4>
         <label id="titleLabel" for="name" class="input-label">Lesson Name</label>
         <input id="titleInput" class="form-control" type="text" name="name" value="{{ lesson_to_reschedule.name }}">
         <label id="locationLabel" for="location" class="input-label">Lesson Location</label>
-        <input id="locationInput" class="form-control" type="text" name="location" value="{{ lesson_to_reschedule.location }}">
+        <input id="locationInput" class="form-control" placeholder = "" onFocus="geolocate()" type="text" name="location" value="{{ lesson_to_reschedule.location }}">
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA1WhiK7CLDk9IkV1MnDB1vVGhbSvOdTl4&libraries=places&callback=initAutocomplete"
```

Comment:
Seems to me you are adding the google maps api more than once. Can you fix this? This is something the console is complaining about too: "You have included the Google Maps JavaScript API multiple times on this page. This may cause unexpected errors"